wcivf/settings/base.py

```diff
@@ -207,8 +209,9 @@
 }
 
 GO_CARDLESS_PAYMENT_NAME = "Democracy Club Donation"
-GO_CARDLESS_PAYMENT_DESCRIPTION = "Helping Democracy Club increase the quantity,"\
-" quality and accessibility of information on election candidates, politicians and democratic processes"
+GO_CARDLESS_PAYMENT_DESCRIPTION = "Helping Democracy Club "\
+"increase the quality of information on elections & the democratic processes"
+
 GOCARDLESS_REDIRECT_URL = "https://democracyclub.org.uk/donate/thanks/"
```

Comment:
After going through the donate process, I'm getting dumped to the corporate site, but you've also added a `/donate/thanks` in this PR. Seems like the _intent_ is that you want people to end up at https://whocanivotefor.co.uk/donate/thanks ?